akd/src/directory.rs

```diff
@@ -219,7 +230,7 @@ impl<S: Storage + Sync + Send, V: VRFKeyStorage> Directory<S, V> {
             .get_root_hash_at_epoch::<_, H>(&self.storage, next_epoch)
             .await?;
 
-        self.storage.log_metrics(log::Level::Info).await;
+        // self.storage.log_metrics(log::Level::Info).await;
```

Comment:
Does it make sense to just remove this commented statement completely?

Reply:
I think it's a great debugging tool but it requires care to use it correctly. Better to manually add/remove in tests.